libraries/DataFlash/DataFlash.cpp

```diff
@@ -50,6 +50,12 @@ const AP_Param::GroupInfo DataFlash_Class::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("_FILE_DSRMROT",  4, DataFlash_Class, _params.file_disarm_rot,       0),
 
+    // @Param: _MAV_BUFSIZE
+    // @DisplayName: Maximum DataFlash MAVLink Backend buffer size (in kilobytes)
+    // @Description: Maximum amount of memory to allocate to DataFlash-over-mavlink.  Values should be some multiple of 212 bytes.
```

Comment:
Please add an `@increment: 212` to reflect this

Reply:
Increment 212/1024 ??? 🤣 😄

Reply:
I'd probably remove the comment about 212 bytes, it's confusing for most users, it will only result in using less memory then you expected (and it won't cause odd behavior if it's not a multiple of 212). It's also worth noting that the default parameter value selected is a slight behavior change as it increases the buffer size.

Reply:
I've removed the comment.

Couldn't slip the behaviour change through, huh?  I couldn't see a nice way of doing otherwise.

Reply:
I'm fine with it actually, I was just noting that it existed :)